cfpb/cfgov-refresh
# Create Media to ContentImage data migration

The new unified `ContentImage` block was created in #4177, leaving the original full-width Media in place until a data migration could be written. This is that data migration.

## Additions

- Data migration to migrate all content that used to go in `FullWidthText` `media` fields into `image_inset` fields (old name, but now refer to the  `ContentImage' block – see the todos below).
- A test for said data migration

## Testing

_Remember: When looking at images from a production dump in the local Wagtail admin, you won't see the actual images, but all the metadata should be there._

1. Open http://localhost:8000/admin/pages/3007/edit/ and see a few of ye olde `media` blockes
1. Pull branch
1. Run migrations
1. Refresh http://localhost:8000/admin/pages/3007/edit/ and see that the `media` blocks have been converted to the new `image` block.
1. Go to http://localhost:8000/admin/pages/11718/edit/ and see that the most recently-created blog with an `image` block is still showing correctly.

## Notes

There is no backward migration from `image_inset` back to `media` in 0117. @chosak agreed that this probably wasn't necessary, but if you disagree with that, let me know.

## Todos

- [x] It's occurring to me now that, while we're here writing a data migration, I could also rename `FullWidthText.image_inset` to just `image`. I'll do that!
- [x] Can also add a regular migration to this that removes `media` entirely.

## Checklist
